.github/workflows/lifecycle-image.yml

```diff
@@ -0,0 +1,24 @@
+name: lifecycle-image
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Get release version
+        id: get_release_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\/v/}
```

Comment:
Minor: might be safer to use the tag name from the `release` object.

Something like `${{ github.event.release.tag_name }}` or `${{ github.event.tag_name }}`, don't know exactly how it's reflected.

Ref: https://developer.github.com/webhooks/event-payloads/#release

Reply:
Giving it a try... we still need to trim off the "v" in the release tag.